### PR TITLE
Update "build" badge in README.md

### PR DESCRIPTION
**Pull request by gregcorbett**

### CODE REVIEW

**Review by tofu-rocketry — COMMENTED** [review]
Does this pick up the default branch's status or just the latest commit?

**Comment by gregcorbett** [comment]
By default, badges display the status of your default branch. - https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # GOCDB
 
-[![Build Status](https://travis-ci.org/GOCDB/gocdb.svg?branch=dev)](https://travis-ci.org/GOCDB/gocdb)
+[![_Continuous Integration](https://github.com/GOCDB/gocdb/actions/workflows/continuous-integration.yml/badge.svg)](https://github.com/GOCDB/gocdb/actions/workflows/continuous-integration.yml)
```
Comment (tofu-rocketry) [inline README.md:3]:
```suggestion
[![_Continuous Integration](https://github.com/GOCDB/gocdb/actions/workflows/continuous-integration.yml/badge.svg)](https://github.com/GOCDB/gocdb/actions/workflows/continuous-integration.yml?query=branch%3Adev)
```

Comment (gregcorbett) [reply README.md:3]:
```suggestion
[![_Continuous Integration](https://github.com/GOCDB/gocdb/actions/workflows/continuous-integration.yml/badge.svg?branch=dev)](https://github.com/GOCDB/gocdb/actions/workflows/continuous-integration.yml?query=branch%3Adev)
```

**Review by tofu-rocketry — APPROVED** [review]

**Pull request merged**